src/interpreter/ByteCode.h

```diff
@@ -256,6 +256,8 @@ class ByteCode {
 
     Opcode opcode() const;
     size_t getSize();
+    virtual const std::string getName() const { return "MISSING NAME"; };
+    virtual void printExtra(size_t pos) { return; };
```

Comment:
I recommend not to use `virtual` functions in ByteCode because the address of vtable is added for each ByteCode instance which increases the overall ByteCode stream size.

Reply:
Okay.
Thank you!